docs/learn/liquidity.md

```diff
@@ -138,17 +138,17 @@ developed by Zeitgeist is based) leave plenty of opportunity for arbitrageurs.
 
 Video tutorials on liquidity pools:
 
--   [Finematics on Liquidity Pools](https://www.youtube.com/watch?v=cizLhxSKrAc)
```

Comment:
When I run prettier on this file (and any other file), it inserts three whitespaces after the dash. Does prettier have a different effect for you?

Reply:
When I run Prettier on my vscode, seems it won't insert 3 whitespaces, a little bit strange.

> When I run prettier on this file (and any other file), it inserts three whitespaces after the dash. Does prettier have a different effect for you?

Reply:
Okay, riddle solved. The prettier integration for my editor uses prettier-1.0.0. My bad!